src/pages/Login.jsx

```
@@ -0,0 +1,303 @@
+          const idRegex = /^(?=.*[A-Za-z])(?=.*?[0-9]).{6,}$/;
+          isValid = idRegex.test(value);
+          message = isValid
+            ? "사용 가능한 아이디 입니다."
+            : "ID를 영문, 숫자 포함 6자 이상 입력해주세요";
+          break;
+        case "pw":
+          const pwRegex =
+            /^(?=.*[A-Za-z])(?=.*\d)(?=.*[$@$!%*#?&])[A-Za-z\d$@$!%*#?&]{8,}$/;
+          isValid = pwRegex.test(value);
+          message = isValid
+            ? "사용 가능한 비밀번호 입니다."
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)